reference-implementation/.eslintrc.json

```diff
@@ -19,7 +19,7 @@
     // Possible errors
     "comma-dangle": ["error", "never"],
     "no-cond-assign": ["error", "except-parens"],
-    "no-console": "error",
+    "no-console": "off",
```

Comment:
Don't turn this off globally. Turn it off inside the file that uses console, using `/*eslint-disable no-console */`